src/inc/sift-events/sift-events.php

```diff
@@ -565,12 +569,17 @@ public static function update_or_create_order( string $order_id, \WC_Order $orde
 		}
 
 		// Determine user and session context.
-		$user_id   = wp_get_current_user()->ID ?? null; // Check first for logged-in user.
+		$user_id   = get_current_user_id() ?? wp_get_current_user()->ID ?? null; // Check first for logged-in user.
```

Comment:
get_current_user_id() definition:
```
function get_current_user_id() {
	if ( ! function_exists( 'wp_get_current_user' ) ) {
		return 0;
	}
	$user = wp_get_current_user();
	return ( isset( $user->ID ) ? (int) $user->ID : 0 );
}


```

As you can see it is using `wp_get_current_user` internally. So I am not sure what this change adds...

Reply:
I'm not sure if it's needed anymore. At one point this made a difference, but I think I addressed that issue in another way.